src/xdocs/google_style.xml

```diff
@@ -462,7 +462,7 @@
                             <td><a name="4.1.2"/><a href="#4.1.2"><img src="images/anchor.png" alt=""/></a></td>
                             <td>
                                 <a
-                                    href="http://checkstyle.sourceforge.net/reports/google-java-style.html#s4.1.2-blocks-k-r-style">4.1.2 Nonempty blocks: K &amp; R style</a>
+                                    href="http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html#s4.1.2-blocks-k-r-style">4.1.2 Nonempty blocks: K &amp; R style</a>
                             </td>
```

Comment:
please show in example how Curly checks are affected by lambda. if so , please create new issue on this.